_docs/hackathon-preperation-checklist.md

```diff
@@ -0,0 +1,43 @@
+# The Week Before
+- [x] testing task list
+
+- Make a team
+- Brainstorm ideas
+    - [link to project-ideas page?]
+- Update your software
+- Get used to AWS?
+    - something to do with getting used to cloud technology
+# Dawn of the Final Day
+blurb about stuff to do day before event
+
+## 24 hours remain
+- stuff to do the day before
+- Charge devices
+- Make a playlist
+- Check-in with group
+
+### What to Bring
+- Laptop(s), Power Cables, USB Cables, Mouse, (Mechanical) Keyboard, HDMI Cable 
+
+- Adapter Cables to full-size HDMI, USB
+- A flash drive (for slides or quick file sharing)
+- (Charged) Phone + Charger
+- Headphones
+- Water bottle
+- Toiletries / Comfy clothes (still can be useful for a single-day event)
+
+Some hardware, like Raspberry Pis and the required components to use them will be provided to you. AWS will be provided to you. Food will be provided, but if you have some dietary restrictions, plan accordingly.
```

Comment:
I fleshed out some of these comments. Note that we will not be providing AWS access (or any other service providers); students must set that up on their own. See changes under "1-2 Weeks Beforehand".
```suggestion
Some hardware, like Raspberry Pis and the required components to use them, will be provided to teams on a first-come, first-served basis if you wish to incorporate them in your project. The devices must be returned to the organizers at the end of the event.

Food will be provided, but if you have some dietary restrictions, plan accordingly. Contact the hackathon organizers for specific questions about dietary needs.
```